src/MoBi.Presentation/Presenter/SelectObjectInTreePresenter.cs

```diff
@@ -68,7 +75,43 @@ public bool IsValidSelection(ObjectBaseDTO selectedDTO)
 
       public void InitTreeStructure(IReadOnlyList<ObjectBaseDTO> entityDTOs)
       {
-         _view.BindTo(entityDTOs);
+         _view.Display(entityDTOs.Select(mapToNode));
```

Comment:
This used to send the dtos directly to the view, which would then call the mapper to create nodes, but that's not the usual pattern. It was just a straight code move.